test/docfx.Test/build/OpsConfigAdapterTest.cs

```diff
@@ -44,5 +44,31 @@ public static async Task AdaptOpsServiceConfig(string url, string expectedJson)
                 JToken.Parse(expectedJson.Replace('\'', '"')),
                 JToken.Parse(actualConfig));
         }
+
+        [Theory]
```

Comment:
The `AdaptOpsServiceConfig` test is skipped via `[SkippableTheory]`, do you know why it is skipped and can it be re-enabled?

If it cannot be re-enabled, remove the test. If it can, enable it and share the `[InlineData]` with the above test using `[MemberData]`.

Reply:
`[SkippableTheory]` is to skip these tests locally since there is no token provide.

Basically, there are 2 kinds of tests, one set is with op token provided, one is not.

Reply:
Try use `[MemberData]` to share the test data.